EnglishFilter/sections/foreign.txt

```diff
@@ -9165,6 +9165,13 @@ brandbuffet.in.th#@#.category-advertorial
 !------------------------------ Vietnam ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76063
+coinurl.net#%#//scriptlet("set-constant", "adsHeight", "15")
+coinurl.net#%#//scriptlet("prevent-window-open")
+coinurl.net##div > [href]
```

Comment:
dangerous rule.

Reply:
I know it's risky but this is enough to block all banners without breaking anything.